### PR TITLE
[INTERNAL] Document Component Preload Generation Configuration

### PR DESCRIPTION
**Pull request by RandomByte**
See https://github.com/SAP/ui5-cli/issues/239

### CODE REVIEW

File: docs/pages/Configuration.md [hunk 1]
```diff
@@ -303,6 +317,20 @@ builder:
         color: blue
 ````
 
+### JSDoc
+You can exclude a projects resources from the JSDoc build process using a list of glob patterns. Matching resources will be ignored by the JSDoc build task.
```
Comment (KlattG) [inline docs/pages/Configuration.md:321]: You can exclude the resources of a project ...

File: docs/pages/Configuration.md [hunk 2]
```diff
@@ -303,6 +317,20 @@ builder:
...
+
+Patterns are always applied relative to the projects virtual **source** directory `/resources/`.
```
Comment (KlattG) [inline docs/pages/Configuration.md:323]: Patterns are always applied relative to the project's  ...

File: docs/pages/Configuration.md [hunk 3]
```diff
@@ -303,6 +317,20 @@ builder:
...
+
+Any general builder excludes (as defined in `builder.resources.excludes`) are applied *after* these excludes.
```
Comment (KlattG) [inline docs/pages/Configuration.md:325]: These excludes are applied *before* any general builder excludes that have been defined in `builder.resources.excludes`.

File: docs/pages/Configuration.md (hunk 1 above)
Comment (matz3) [reply docs/pages/Configuration.md]: See https://github.com/SAP/ui5-tooling/pull/258

File: docs/pages/Configuration.md (hunk 2 above)
Comment (matz3) [reply docs/pages/Configuration.md]: See https://github.com/SAP/ui5-tooling/pull/258

File: docs/pages/Configuration.md (hunk 3 above)
Comment (matz3) [reply docs/pages/Configuration.md]: See https://github.com/SAP/ui5-tooling/pull/258

**Comment by matz3** [comment]
Extracted JSDoc related changes to https://github.com/SAP/ui5-tooling/pull/258

**Review by RandomByte — COMMENTED** [review]
Fine from my side

File: docs/pages/Configuration.md [hunk 4]
```diff
@@ -301,6 +301,33 @@ builder:
...
+
+For projects of type `application` a `Component-preload.js` bundle is generated by default.  
+You can override this default behavior by defining a `componentPreload` configuration.
+
+For projects of type `library` there is no Component Preload created by default.  
+However you can define a `componentPreload` configuration to create Component Preload bundles. Those will be created in addition to the `library-preload.js` bundle.
+
+There are two ways of including components, which can also be used in combination with each other:
+
+- The `paths` option takes one or multiple patterns. For every matched file a separate `Component-preload.js` will be generated. Patterns are always applied relative to the projects virtual source directory `/resources/`
```
Comment (codeworrior) [inline docs/pages/Configuration.md:314]: ... relative to the project's ...

**Review by codeworrior — COMMENTED** [review]
LGTM.

File: docs/pages/Configuration.md (hunk 4 above)
Comment (matz3) [reply docs/pages/Configuration.md]: Oh, yeah thanks. Done

**Review by KlattG — APPROVED** [review]
LGTM

**Review by matz3 — APPROVED** [review]

**Pull request merged**